### ISSUE TITLE
Round elapsed values below 1ms to one decimal place

### ISSUE THREAD

**Issue by agneevX** [AdguardTeam/AdGuardHome#4543]
### Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Problem Description

Cached queries often take sub-1ms to process. The UI shows such values in two decimal places.

![image](https://user-images.githubusercontent.com/19761269/166137964-9a313545-b534-416e-971e-712cf28d6b63.png)

The extra precision is not really needed and it would otherwise look better and cleaner.

### Proposed Solution

Round values below 1ms to one decimal place. E.g. `0.45 ms` to `0.5 ms`.

**Comment by fvdm** [comment]
I see cache responses around 0.08 ms. Reducing the number of decimals would make those look like 0.0 ms as in 'no time' or rounded up to 0.1 which both would be incorrect.

Also it is an indication of how well my box is performing. When I see a lot of 0.45's I know that something is consuming a lot of resources. Personally I like it the way it is now.

**Comment by agneevX** [comment]
@fvdm Guess a special concession could be made for you ;)

**Comment by ainar-g** [comment]
On the one hand, it would indeed look better on the UI.  But on the other, I'm not a big fan of hiding information.  Perhaps the top-level log should be rounded, but the full information about the request should contain a more precise reading.  We'll see.

**Comment by fvdm** [comment]
@ainar-g Yeah. Maybe an precision option in the Query Log settings for the UI and the numbers with their two-decimals in the API response.

**Comment by ainar-g** [comment]
Added a `feature request` label to avoid the bot.
